rubin_sim/maf/run_comparison/summary_plots.py

```diff
@@ -83,6 +84,15 @@ def normalize_metric_summaries(
     # Get rid of duplicate metrics and runs
     summary = summary.T.groupby("metric").first().T.groupby("run").first()
 
+    # And now create a line just for "baseline" --
+    # if baseline_run is >1, this is created from the median values per metric of those runs
+    if isinstance(baseline_run, list):
```

Comment:
Rather than checking whether `baseline_run` is a list, it would make the code more versatile to check if `summary.loc[baseline_run] `returns a single value or multiple values. Sometimes it will make more sense for a user to supply other sorts of collections, e.g. a `set`, `tuple`, or a `pandas.Series`, and supporting this would not make the code any more complex.

Reply:
Oh good idea ... although, this might be problematic as it is .. 
```  if len(summary.loc[baseline_run]) > 1:
/var/folders/tr/bjzqy0h17kx31fhndw6050d80000gn/T/ipykernel_16607/116705069.py:65: FutureWarning: Passing a set as an indexer is deprecated and will raise in a future version. Use a list instead.
```

Reply:
So: 
making the list a set instead gives me a future warning that it won't be supported in the future
making the list a tuple gives me key errors if the tuple is more than one element long (which I don't understand but may be a panda + me thing? it does seem like the same thing that works as a list, if the list is converted to a tuple, no longer works)
making the list a pd.Series using only the list does work

I'll make the adjustment because you're right that's a better and more flexible way to do it, but it's worth noting that set will work for now but fail later, and a tuple will not generally work as far as I can tell.  But at least the errors will be clearly pandas-related errors, which will hopefully lead people to realize they can't reference contents of a data frame in that way.

Reply:
Also .. this may be me not knowing something about pandas, but if you provide only a string, then the summary.loc[baseline_run] returns a series, which has different kinds of length criteria than if you provided a list which then returns a data frame. 
Is there a benefit to checking if the returned value from "summaries.loc[baseline_run]" is a Series or a Dataframe, and then also checking the length of the Dataframe? Vs. just checking if baseline_run is a string (and if it's anything other than a string, assuming you will just go ahead and take the median .. if it's only a single element, taking the median would be the same as taking the row itself).